ztools/x509/cert_pool.go

```diff
@@ -77,14 +77,13 @@ func (s *CertPool) AddCert(cert *Certificate) {
 }
 
 // AppendCertsFromPEM attempts to parse a series of PEM encoded certificates.
-// It appends any certificates found to s and returns true if ALL certificates
```

Comment:
If there were other changes, that's fine. I just want to make sure this function keeps our modified semantics.